doc/spire_server.md

```diff
@@ -94,10 +94,10 @@ This may be useful for templating configuration files, for example across differ
 | `cache_reload_interval`   | The amount of time between two reloads of the in-memory entry cache. Increasing this will mitigate high database load for extra large deployments, but will also slow propagation of new or updated entries to agents. | 5s                                 |
 | `events_based_cache`      | Use events to update the cache with what's changed since the last update. Enabling this will reduce overhead on the database.                                                                                          | false                              |
 | `prune_events_older_than` | How old an event can be before being deleted. Used with events based cache. Decreasing this will keep the events table smaller, but will increase risk of missing an event if connection to the database is down.      | 12h                                |
-| `sql_transaction_timeout` | Maximum time an SQL transaction could take, used by the events based cache to determine when an event id is unlikely to be used anymore.                                                                               | 24h                                |
```

Comment:
Could we also keep this option around for a bit longer as deprecated? It's not required, since it's experimental, but it would be nice to make the upgrade path easier for users.

Reply:
Added the option back in to the code.

Reply:
Yes, thank you both. I might have missed this rename without this comment thread here. The original name was confusing in that it's actually a lookback period for events (even if it was originally set to match a postgresql default transaction timeout). The new name is much better!

Also, it's really good to know that we're settling on 5 minutes now to guide us adjusting the old configurable.

Reply:
It's not technically a timeout. It's a lookback window. Perhaps we name it `event_lookback_window` (or something like that) documented as "maximum age of events to query" (or something like that)?

Reply:
@zmt That is definitely more accurate as to whats happening. Though I think timeout is a more standard naming convention and might be more intuitive for users.